module/mouse_gutbrain_de_independent_guide.qmd

```diff
@@ -218,21 +226,21 @@ Here are the URLs for all the possible comparisons you can examine with this dat
 
 **Comparing gene expression between ASD and control mice**
 
-Both brain regions: [https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol.csv](https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol.csv)
+Both brain regions: <https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol.csv>
 
-Prefrontal cortex only:  [https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_prefrontalcortex.csv](https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_prefrontalcortex.csv)
+Prefrontal cortex only: <https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_prefrontalcortex.csv>
 
-Striatum only: [https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_striatum.csv](https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_striatum.csv)
+Striatum only: <https://genomicseducation.org/data/mouse_gutbrain_de_autismVcontrol_in_striatum.csv>
 
 **Comparing gene expression between prefrontal cortex and striatum**
 
-All mice: [https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype.csv](https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype.csv)
+All mice: <https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype.csv>
 
-Only ASD mice: [https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_ASDmice.csv](https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_ASDmice.csv)
+Only ASD mice: <https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_ASDmice.csv>
 
-Only control mice: [https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_controlmice.csv](https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_controlmice.csv)
+Only control mice: <https://genomicseducation.org/data/mouse_gutbrain_de_tissuetype_in_controlmice.csv>
 
-#### Arranging dataset based on padj 
+#### Arranging dataset based on padj
 
 You may be interested in identifying the genes that have been differentially expressed between your two groups (in this case, ASD mouse brains and control mouse brains). You can use the `arrange` command to sort the results to put the smallest padj values first.
```

Comment:
might add how to interpret. For example, notice how the top log2foldchange value is -25! This means xyz..